test/http/doc_examples.cpp

```diff
@@ -307,6 +311,50 @@ class doc_http_samples_test
         BEAST_EXPECTS(! ec, ec.message());
     }
 
+    void
+    doConstAndMutableBody()
+    {
+        test::pipe c{ios_};
+
+        std::array<char, 15> const body {{"Hello, world!\n"}};
+        {
+            request<const_body<std::array<char, 15>>> req;
+            req.version = 11;
+            req.method(verb::put);
+            req.target("/");
+            req.body = body;
+            req.prepare_payload();
+            write(c.client, req);
```

Comment:
Nothing to do here except to note that people using `std::array` need to be careful because the entire array will be written out no matter how big the string is.